test/Microsoft.AspNetCore.SignalR.Client.Tests/TestConnection.cs

```diff
@@ -136,5 +141,48 @@ private byte[] FormatMessageToArray(byte[] message)
                 Closed?.Invoke(ex);
             }
         }
+
+        public IDisposable OnReceived(Func<byte[], object, Task> callback, object state)
+        {
+            var receiveCallBack = new ReceiveCallBack(callback, state);
+            _callbacks.Add(receiveCallBack);
+            return new Subscription(receiveCallBack, _callbacks);
+        }
+
+        private class ReceiveCallBack
+        {
+            public readonly Func<byte[], object, Task> CallBack;
```

Comment:
make it a getter (and fix casing)

Reply:
This should actually be private since I've added the InvokeAsync method